install.sh

```diff
@@ -39,6 +39,10 @@ MACOS_LATEST_SUPPORTED="10.15"
 # TODO: bump version when new macOS is released
 MACOS_OLDEST_SUPPORTED="10.13"
 
+# For Homebrew on Linux
+required_ruby_version=2.6
+required_glibc_version=2.13
```

Comment:
Uppercase these and use `"` to match style of variables above. Also: might be worth linking to the document/code that defines these requirements.